array-methods/map.js

```diff
@@ -0,0 +1,11 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+const priceObjects = prices.map((price, salePrice) => {
+  salePrice = price / 2;
+  return { price, salePrice };
+});
+console.log('Price objects:', priceObjects);
+
+const formattedPrices = prices.map((price) => {
+  return `$${price}`;
+});
```

Comment:
You need to make use of hte `toFixed` method here, so that there's only 2 numbers after the decimal point.